docs/sources/next/extensions/explore.md

```diff
@@ -95,14 +87,6 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-dotenv</h4>
         <p>Load env vars from a .env file</p>
     </a>
-    <a href="https://github.com/szkiba/xk6-ts" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
Extension owner changed this is why name changed from "szkiba/xk6-ts" to "grafana/xk6-ts"